lnrpc/routerrpc/router.swagger.json

```diff
@@ -1356,6 +1356,15 @@
         }
       }
     },
+    "routerrpcHTLCEndorsement": {
+      "type": "string",
+      "enum": [
+        "ENDORSEMENT_UNKNOWN",
```

Comment:
Can be `ENDORSMENT_DEFAULT` as endorsement it's not unknown, just picked up by default by implem.